test/tests/functional/pbs_reservations.py

```diff
@@ -789,9 +789,14 @@ def test_future_resv_conflicts_running_job(self):
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
 
+        if self.mom.is_cpuset_mom():
+            select_val = vnode_val
+        else:
+            select_val = '1:ncpus=1'
+
```

Comment:
Why don't you combine this if statement with the if self.mom.is_coupsaet_mom() if statement above

Reply:
I've made the change